### PR TITLE
data: jyotir07

### PR DESCRIPTION
**Pull request by jyotir07**
## Changes proposed

My first contribution to opensource

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5362"></a>

### CODE REVIEW

File: data/jyotir07.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "links": [
+    {
+      "name": "GitHub: Let's collaborate",
+      "url": "https://github.com/jyotir07",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/<Your Twitter Username>",
+      "icon": "FaTwitter"
```
Comment (Pradumnasaraf) [inline data/jyotir07.json:26]: Add your Twitter GitHub usernames, if not, remove the section.

```suggestion
      "name": "Twitter: Follow me",
      "url": "https://twitter.com/<Your Twitter Username>",
      "icon": "FaTwitter"
```

File: data/jyotir07.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    {
+      "name": "LinkedIn: Let's connect",
+      "url": "Your Linkedin Profile URL",
```
Comment (Pradumnasaraf) [inline data/jyotir07.json:30]: Add your LinkedIn URL here. If you don't have to remove this block
```suggestion
      "url": "Your Linkedin Profile URL",
```

File: data/jyotir07.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  "tags": ["WebDev", "tag2", "tag3"],
+
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/<Your Twitter Username>" },
+    { "icon": "FaGithub", "url": "https://github.com/<Your GitHub Username>" }, //for mini icons
+    {
+      "icon": "FaLinkedin",
+      "url": "Your Linkedin Profile URL"
+    }
+  ],
```
Comment (Pradumnasaraf) [inline data/jyotir07.json:15]: Add your Twitter, GitHub and LinkedIn usernames, if not, remove the section and also remove the comments (`//for mini icons`)


```suggestion
  "socials": [
    { "icon": "FaTwitter", "url": "https://twitter.com/<Your Twitter Username>" },
    { "icon": "FaGithub", "url": "https://github.com/<Your GitHub Username>" }, //for mini icons
    {
      "icon": "FaLinkedin",
      "url": "Your Linkedin Profile URL"
    }
  ],
```

File: data/jyotir07.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Jyotiraditya Singh",
+  "type": "personal",
+  "bio": "**Your Bio**",
```
Comment (Pradumnasaraf) [inline data/jyotir07.json:4]: Add a bio

```suggestion
  "bio": "**Your Bio**",
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

There are some missing elements in the JSON file please update them to make the PR ready for merge

**Comment by SaraJaoude** [comment]
We look forward to seeing your Profile on LinkFree, but as there is no recent activity from you to make the changes suggested I am going to close this pull request.

We hope you will raise a new pull request to create your Profile and when you do, you might want to take a look at the following Docs which will help with the suggested changes the maintainers have proposed:

Quickstart: https://linkfree.eddiehub.io/docs/quickstart

Editing your Profile: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Pull request closed without merging**